index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Star +</title>
+    <link rel="stylesheet" href="./css/style.css"/>
+    <link 
+        rel="stylesheet" 
+        href="./css/tablet.css"
+        media="screen and (min-width: 768px)"/>
+    <link 
+        rel="stylesheet" 
+        href="./css/desktop.css"
+        media="screen and (min-width: 1024px)"/>
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300;500;700&family=Sen&display=swap" rel="stylesheet"></head>
+<body>
+    <section class="hero-header"> 
+        <div class="hero-text">
+            <img src="./img/star-logo.svg" alt="Star + Logo" class="star-logo"/>
+            <h2 class="description">Más de lo que te gusta.</h2>
+            <div class="primary-button">
+                <button class="primary-button combo-button">QUIERO COMBO+</button>
+            </div>
+            <a href="/" class="anchor-subscription">Suscribirme solo a Star+</a>
+        </div>
+    </section>
```

Comment:
```suggestion
    <section class="hero"> 
        <div class="hero__content">
            <img class="hero__image" src="./img/star-logo.svg" alt="Star + Logo" "/>
            <h2 class="hero__title">Más de lo que te gusta.</h2>
            <div class="hero__button-container">
                <button class="hero__button">QUIERO COMBO+</button>
            </div>
            <a class="hero__link" href="/" >Suscribirme solo a Star+</a>
        </div>
    </section>
```

Please check this article https://scalablecss.com/bem-nesting-grandchild-elements/